README.md

```diff
@@ -220,6 +236,7 @@ information how to work with them.
 | Project                         | Status                           | Ideas Pages                                         |
 | -------                         | ------                           | -----------                                         |
 | [AstroPy]                       | Applying with Open Astronomy     | http://openastronomy.org/gsoc/gsoc2018/#/projects   |
+| [Cantera]                       | Applying under NumFOCUS umbrella |                                                     |
```

Comment:
this might have broken the link. Markdown is case sensitive.

Reply:
@kain88-de It doesn't seem to have broken the link, either on GH or in Atom's MD preview. Should I still fix it? Also, FYI, we will have our mentors and admin signing up on the Google form you sent in the next few days, and we plan to keep updating the ideas list as well 😃 Thank you!

Reply:
Yeah I noticed myself that the link still works.  Thanks for the pr again.